### PR TITLE
scripts: use venv for python dependencies

### PR DESCRIPTION
**Pull request by NikitaZotov**

### CODE REVIEW

**Review by MksmOrlov — CHANGES_REQUESTED** [review]
image

**Review by MksmOrlov — APPROVED** [review]
Don't forget about set_vars

File: .github/workflows/install.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04, macos-latest]
+        os: [macos-13, macos-14, ubuntu-20.04, ubuntu-22.04]
```
Comment (FallenChromium) [inline .github/workflows/install.yml]: I don't think we need to check our install mechanism twice on macOS

File: scripts/set_vars.sh [hunk 2]
```diff
@@ -8,8 +8,8 @@ export PLATFORM_PATH="${PLATFORM_PATH:-${ROOT_PATH}}"
 export CONFIG_PATH="${CONFIG_PATH:-${PLATFORM_PATH}/ostis-web-platform.ini}"
 export REPO_PATH="${REPO_PATH:-${PLATFORM_PATH}/repo.path}"
 
-export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/ostis-ai/sc-machine.git}"
-export SC_MACHINE_BRANCH="${SC_MACHINE_BRANCH:-main}"
+export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/NikitaZotov/sc-machine.git}"
```
Comment (FallenChromium) [inline scripts/set_vars.sh]: I think we shouldn't merge custom repo URL to `develop`

Comment (FallenChromium) [reply scripts/set_vars.sh]: I believe you did this to check CI, change this after merging https://github.com/ostis-ai/sc-machine/pull/384 and https://github.com/ostis-ai/sc-web/pull/169 please

**Review by FallenChromium — APPROVED** [review]

File: scripts/set_vars.sh (hunk 2 above)
Comment (NikitaZotov) [reply scripts/set_vars.sh]: > I think we shouldn't merge custom repo URL to `develop`

There is separated commit for this to test ci

**Pull request merged**